### PR TITLE
WFLY-8415 WFLY-8134 - Testsuite refactoring

### PR DESCRIPTION
**Pull request by ctomc**
- fix https://issues.jboss.org/browse/JBEAP-8875
- remove changeIPs.xsl transformation
- use CLI plugin

### CODE REVIEW

**Comment by rhusar** [comment]
I really like the CLI approach, opened https://issues.jboss.org/browse/WFLY-8303 to remove XSLTs in clustering testsuite.

**Comment by ctomc** [comment]
@rhusar yeah, I prepared most stuff to move clustering TS to this, but it needs much more work to work in all scenarios. mostly xsite config tests need more attention.

File: testsuite/integration/basic/src/test/config/arq/arquillian.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <container qualifier="jboss" default="true">
         <configuration>
             <property name="jbossHome">${basedir}/target/jbossas</property>
-            <property name="javaVmArguments">${server.jvm.args} -Djboss.inst=${basedir}/target/jbossas -Dtest.bind.address=${node0}</property>
+            <property name="javaVmArguments">${server.jvm.args} -Djboss.inst=${basedir}/target/jbossas</property>
```
Comment (kabir) [inline testsuite/integration/basic/src/test/config/arq/arquillian.xml]: @rsvoboda @kwart does QE use node0 to change the bind address in scripts?

**Comment by kabir** [comment]
@darranl @kwart This moves the elytron testsuite from testsuite/elytron to testsuite/integration/elytron. Does that affect work going on in ladybird?

File: testsuite/integration/iiop/src/test/config/arq/arquillian.xml [hunk 2]
```diff
@@ -26,14 +26,14 @@
         <container qualifier="iiop-server" default="false">
             <configuration>
                 <property name="jbossHome">${basedir}/target/jbossas-iiop-server</property>
-                <property name="javaVmArguments">${server.jvm.args}</property>
+                <property name="javaVmArguments">${server.jvm2.args}  -Djboss.socket.binding.port-offset=100</property>
```
Comment (kabir) [inline testsuite/integration/iiop/src/test/config/arq/arquillian.xml]: Why this rename, are you sure QE doesn't use this?

**Comment by darranl** [comment]
At the time this is merged we will need to verify if it also merges cleanly with the ongoing work in ladybird and check if any other engineers have work in progress that could be broken by changing the locations of the files.

File: testsuite/integration/basic/src/test/config/arq/arquillian.xml (hunk 1 above)
Comment (kwart) [inline testsuite/integration/basic/src/test/config/arq/arquillian.xml]: QE uses the `node0` to configure the bind address. Nonetheless it seems the `"test.bind.address"` is not used in the testsuite, so it should be fine to remove this one from the configuration.

**Comment by kwart** [comment]
@kabir, @ctomc @darranl IMO, it's not the right time to move the Elytron TS module now. There is ongoing Elytron related test development into the AS TS. QE team follows the agreement and sends changes primarily to ladybird branch.

We should either wait with the move for a better time (once ladybird is not used anymore). Or this PR should be sent to ladybird and we should discuss an official date for merging it - so the affected (test-)developers have chance to prepare for the change (e.g. to send their changes to be merged before this one).

File: testsuite/integration/basic/src/test/config/arq/arquillian.xml (hunk 1 above)
Comment (rsvoboda) [inline testsuite/integration/basic/src/test/config/arq/arquillian.xml]: +1 to remove

File: testsuite/integration/iiop/src/test/config/arq/arquillian.xml (hunk 2 above)
Comment (rsvoboda) [inline testsuite/integration/iiop/src/test/config/arq/arquillian.xml]: It's "implementation detail", we do not depend on this property name. So safe to rename.

**Comment by rsvoboda** [comment]
Please create this PR on ladybird branch as it affects parts related to security/elytron.
  or 
Wait till Ladybird is shutdown - should be closed when we have ER1 tag

**Comment by darranl** [comment]
We are transitioning between ER1 and ER2 - not at the point ER1 is tagged.

**Comment by rsvoboda** [comment]
PR on ladybird branch is probably the best option

File: testsuite/integration/iiop/src/test/config/arq/arquillian.xml (hunk 2 above)
Comment (ctomc) [inline testsuite/integration/iiop/src/test/config/arq/arquillian.xml]: it is not rename as such, there are two sets of properties for two servers that are started at same time.
now that xslt is not used to modify the binding props we need them separtly

**Comment by rsvoboda** [comment]
Downstream runs fail with `testsuite/integration/elytron/pom.xml: Duplicated tag: 'artifactId' (position: START_TAG seen ...</artifactId>\n <artifactId>..`

**Comment by rsvoboda** [comment]
Downstream is green again, some GH woodoo

**Pull request merged**